### PR TITLE
packit: add Fedora 38 testing and release

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: packit.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
     targets:
       - fedora-36
```
Comment (martinpitt) [inline packit.yaml:22]: Let's please drop fedora-36. We have traditionally only supported two parallel releases, there's too much overhead otherwise.

**Review by martinpitt — CHANGES_REQUESTED** [review]
I'll trigger f-38 to see how it works out.

File: packit.yaml (hunk 1 above)
Comment (jelly) [reply packit.yaml:22]: Ok, I checked and Fedora 36 is supported till F 35. So do we stop releasing and testing or only testing?

**Comment by jelly** [comment]
> I'll trigger f-38 to see how it works out.

But we have no Fedora 38 image yet? I can't build one yet as the cloud image is not yet out. 

https://download.fedoraproject.org/pub/fedora/linux/releases/38/Cloud/x86_64/images/

File: packit.yaml (hunk 1 above)
Comment (martinpitt) [reply packit.yaml:22]: We stop both. (Logically, stop releasing first and then testing, but we can stop both at the same time in packit)

**Comment by martinpitt** [comment]
> But we have no Fedora 38 image yet? I

Ah yes, of course :see_no_evil:

**Comment by jelly** [comment]
F38 succeeded here, so re-pushed with the dropping of F36 https://artifacts.dev.testing-farm.io/3eeaf539-248b-457b-9e6c-62b8c4654e43/

**Review by martinpitt — APPROVED** [review]
Nice!

**Pull request merged**